src/tests/components/GoogleLogin.test.js

```diff
@@ -36,4 +35,12 @@ describe('MapDispatchToProps', () => {
       expect(dispatch).toHaveBeenCalled();
     });
   });
+
+  describe('MapStateToProps', () => { 
+    const state = {modalState: { component: 'register', modalShow: true }}
+    const expected = { component: 'register'}
+    it('returns the modal state', () => {
+      expect(mapStateToProps(state)).toEqual(expected)
+    });
+});
   
```

Comment:
@abulojoshua1 I've added it but it's not reflecting for some reason. line `46`.

Reply:
Ok, it works now. We deleted the empty line and pushed again, twice. I still don't understand how it got fixed. @BrianSerem helped.